openlane/Makefile

```diff
@@ -18,10 +18,34 @@ BLOCKS = $(shell find * -maxdepth 0 -type d)
 CONFIG = $(foreach block,$(BLOCKS), ./$(block)/config.tcl)
 CLEAN = $(foreach block,$(BLOCKS), clean-$(block))
 
+# OpenLane Image
 OPENLANE_TAG ?= 2021.11.23_01.42.34
-OPENLANE_IMAGE_NAME ?= efabless/openlane:$(OPENLANE_TAG)
-OPENLANE_BASIC_COMMAND = "cd $(PWD)/../openlane && flow.tcl -design ./$* -save_path .. -save -tag $* -overwrite"
-OPENLANE_INTERACTIVE_COMMAND = "cd $(PWD)/../openlane && flow.tcl -it -file ./$*/interactive.tcl"
+export OPENLANE_IMAGE_NAME ?= efabless/openlane:$(OPENLANE_TAG)
+
+# OpenLane Root
+export OPENLANE_ROOT ?=
+
+# PDK Root
+export PDK_ROOT ?=
+
+# Caravel Root
+export CARAVEL_ROOT ?= $(shell pwd)
+
+# Management Area Root
+export MGMT_AREA_ROOT ?= $(shell pwd)/mgmt_core_wrapper 
+
+# Parent Folder
+export PARENT_FOLDER = $(shell realpath $(PWD)/..)
+
+# PDK setup configs
+export THREADS ?= $(shell nproc)
+export STD_CELL_LIBRARY ?= sky130_fd_sc_hd
+export SPECIAL_VOLTAGE_LIBRARY ?= sky130_fd_sc_hvl
+export IO_LIBRARY ?= sky130_fd_io
+export PRIMITIVES_LIBRARY ?= sky130_fd_pr
+export SKYWATER_COMMIT ?= c094b6e83a4f9298e47f696ec5a7fd53535ec5eb
+export OPEN_PDKS_COMMIT ?= 27ecf1c16911f7dd4428ffab96f62c1fb876ea70
+export INSTALL_SRAM ?= enabled
```

Comment:
Is it possible to have standard bool evaluations for bool flags? So `INSTALL_SRAM` is either `true` or `false`

Reply:
you can pass that along to open_pdks

Reply:
It's out of the scope for this PR for sure but is it possible to change the `open_pdks` as well? For me it makes using the makefile more transparent, but happy if there's no agreement to this

Reply:
I would say it's even more important since the Makefiles on this level expect `INSTALL_SRAM` to be `yes` or `no` and the Makefiles in the `mgmt_core_wrapper` expect `INSTALL_STAM` to be `enabled` or `disabled`

Reply:
I don't disagree with you. It's just a change that encompasses more tools than I have the bandwidth to juggle at the moment.

Reply:
No worries, let me know if there's anything I can do to help 😄